src/server/routes.js

```diff
@@ -58,18 +60,22 @@ app.post("/inventory/product", function (req, res) {
   const document = req.body;
 
   if (
-    !document.product_id ||
-    !document.style_id ||
-    !document.size ||
-    !document.quantity
+    document.product_id === undefined ||
+    document.style_id === undefined ||
+    document.size === undefined ||
+    document.quantity === undefined
   ) {
     res.sendStatus(400);
     return;
   }
 
   controller
     .write(document)
-    .then(function (result) {
+    .then(async function (result) {
+      const key = `/inventory/${document.product_id}/${document.style_id}`;
```

Comment:
**Annotation**: Key in proxy server is stored as an url. Here we need find the key, which must follow the same format.